djoser/views.py

```diff
@@ -54,6 +54,22 @@ class RegistrationView(generics.CreateAPIView):
     permission_classes = (
         permissions.AllowAny,
     )
+    _users = None
+
+    def create(self, request, *args, **kwargs):
+        try:
+            email_users = self.get_email_users(request.data.get('email'))
+            for user in email_users:
+                serializer = self.get_serializer(instance=user)
+                if settings.SEND_REREGISTRATION_EMAIL:
+                    self.send_reregistration_email(user)
+                headers = self.get_success_headers(serializer.data)
+            if not settings.REREGISTRATION_SHOW_RESPONSE and email_users:
+                return Response(serializer.data, status=status.HTTP_201_CREATED, headers=headers)
+        except User.DoesNotExist:
+            pass
```

Comment:
I'm wondering if this `try ... except ...` block could not be replaced with some smart `if` statement, since you are not doing anything with the exception.